### PR TITLE
New package: DuckStudio.FufuTools version 1.3.2

### PR DESCRIPTION
**Pull request by CoolPlayLin**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/157822)

### CODE REVIEW

**Comment by DuckDuckStudio** [comment]
@Luna-Grace @DuckDuckStudio Need Attention.

File: manifests/d/DuckStudio/FufuTools/1.3.2/DuckStudio.FufuTools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageVersion: 1.3.2
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-06-13
+Installers:
+- Architecture: x86
```
Comment (DuckDuckStudio) [inline manifests/d/DuckStudio/FufuTools/1.3.2/DuckStudio.FufuTools.installer.yaml:14]: 程序是x64架构，如果出错请尝试修改这里。

**Review by DuckDuckStudio — APPROVED** [review]

**Review by Luna-Grace — APPROVED** [review]

**Review by DuckDuckStudio — CHANGES_REQUESTED** [review]

**Review by DuckDuckStudio — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 885.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> Trojan:Win32/Wacatac.B!ml

(Automated response - build 885.)

**Comment by DuckDuckStudio** [comment]
@DuckDuckStudio @Luna-Grace
Links that may be useful:
- [Microsoft Security Intelligence](https://www.microsoft.com/en-us/wdsi/submission/febbb5ed-6252-489e-bffb-956c0c8f6732) -  No result yet.
- [Virustotal](https://www.virustotal.com/gui/file/79f3c2bd9d63035d52a1b6f3884d24b35d629098c2c4398d2d924b04c6018f39?nocache=1) - 2 of 53 security vendors marked the installer file as malicious. (`Trojan.Win32.**Pyinstaller**` by _Ikarus_ ; `Win.MxResIcn.Heur.Gen` by _MaxSecure_)

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 885.)

**Comment by stephengillie** [comment]
> > It may take me a long time to reply, I will look this PR again around 20:00 (UTC+8).

Thank you again for your patience with our processes.

**Comment by stephengillie** [comment]
Manual Validation ended with:
> Trojan:Win32/Wacatac.B!ml Trojan:Win32/Sabsik.TE.A!ml Trojan:Win32/Wacatac.H!ml

(Automated response - build 886.)

**Comment by DuckDuckStudio** [comment]
> Manual Validation ended with:
> 
> > Trojan:Win32/Wacatac.B!ml Trojan:Win32/Sabsik.TE.A!ml Trojan:Win32/Wacatac.H!ml
> 
> (Automated response - build 886.)

Can you tell me which files after installation have these problems?

**Comment by stephengillie** [comment]
> > Manual Validation ended with:
> > > Trojan:Win32/Wacatac.B!ml Trojan:Win32/Sabsik.TE.A!ml Trojan:Win32/Wacatac.H!ml
> > 
> > 
> > (Automated response - build 886.)
> 
> Can you tell me which files after installation have these problems?

Not easily - I would have to monitor the VM and stop it right before it sends the message. Please work with @CoolPlayLin to find this information from his validation environment. Or feel free to create your own local or cloud-based VM for testing.

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 886.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> Trojan:Win32/Wacatac.B!ml Trojan:Win32/Wacatac.H!ml

(Automated response - build 886.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 886.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> Trojan:Win32/Wacatac.B!ml

(Automated response - build 886.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> Trojan:Win32/Wacatac.B!ml Trojan:Win32/Sabsik.FL.A!ml

(Automated response - build 886.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 886.)

**Comment by stephengillie** [comment]
Verify:
Unfortunately, the VM isn't configured to display the application correctly: 
image

image
Internet Translation "FuFu Toolbox"

Hi @CoolPlayLin and @DuckDuckStudio, 

Some applications reporeted missing DLLs , and I wanted to check with you, if these errors mean a dependency is needed. If they're not meant to be run, then it's okay. Otherwise, this PR is ready to merge.  
image
image

**Comment by DuckDuckStudio** [comment]
See DuckDuckStudio/Fufu_Tools#70

**Comment by stephengillie** [comment]
Thanks for the clarification. 

Once the dependency is known, adding it is easy. There are several ways to add a dependency. Here is the one I usually use: at the root level, if the dependency were `Microsoft.VCRedist.2015+.x64`, it would be:

```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by CoolPlayLin** [comment]
@DuckDuckStudio All suggestion seems to be done. BTW, I would like to include DuckStudio.FufuTools into my automations.

**Comment by DuckDuckStudio** [comment]
> @DuckDuckStudio All suggestions seemed to be done. BTW, I would like to include DuckStudio.FufuTools into my automations.

I think it is better to create the PR by myself in the future, which can solve the problem directly.
Of course, if you want to include it in your automation, that'll be okay.

**Comment by DuckDuckStudio** [comment]
@stephengillie Can you reverify the installer?🙏🏼 It shouldn't.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-06-26 04:47:39.780 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.40.33810.0

(Automated response - build 886.)

**Comment by CoolPlayLin** [comment]
@DuckDuckStudio It should be caused by a mismatch dependency

**Comment by DuckDuckStudio** [comment]
This does not seem to be an issue with my installer, I have tested my installer in [#156017 (comment)](https://github.com/microsoft/winget-pkgs/pull/156017#issuecomment-2166782275).

**Comment by stephengillie** [comment]
Verify:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**